### DISCUSSION TITLE
Notify plugin behaving strangely

### DISCUSSION THREAD

**Discussion by elali** [quasarframework/quasar#10153] [General - Components / Directives / etc]
Hi
Not sure if I managed to mess something up somewhere, but I am having some small display issues on the notify plugin. The text seem to wrap even on short sentences (for example a 3 word sentence wraps to one word per line). When the notification fades, it jumps the the corner of the screen and fades from there.

**Comment by MilosPaunovic** [comment]
A code sample of any kind would help in debugging this.

**Reply by elali** [reply]
I just used the normal example from the documentation. I had been playing around with it for a bit before posting here, then afterwards thought about the style sheet. I have been supplied one that the client wants me to use, and it seems this is what is causing the problem. So I don't think there is anything anyone could help with? I am going to try and isolate the class that is causing the issue. Thank you for your response.
